web/styles/rendered_markdown.css

```diff
@@ -721,6 +721,36 @@
             font-size: 1.2em;
             /* 5px at 16.8px (1.2 * 14px) */
             margin-top: -0.2976em;
+            /* We set the markdown link colors here so
+               that the ellipsis takes it on truncated
+               lines. The ellipsis will not take an
+               underline even if we specify one, so
+               that is deliberately omitted here. */
+            color: var(--color-markdown-link);
+
+            &:hover {
+                color: var(--color-markdown-link-hover);
+            }
+
+            a {
```

Comment:
We should make sure this has a class attached to avoid unnecessary work in browser implementations.

Reply:
Quite right. Added the mechanism to do that bit of post-processing, as well as tests to maintain coverage.